src/AppInstallerRepositoryCore/SQLiteWrapper.cpp

```diff
@@ -126,7 +126,10 @@ namespace AppInstaller::Repository::SQLite
     {
         AICLI_LOG(SQL, Info, << "Opening SQLite connection: '" << target << "' [" << std::hex << static_cast<int>(disposition) << ", " << std::hex << static_cast<int>(flags) << "]");
         int resultingFlags = static_cast<int>(disposition) | static_cast<int>(flags);
-        THROW_IF_SQLITE_FAILED(sqlite3_open_v2(target.c_str(), &m_dbconn, resultingFlags, nullptr));
+        if (sqlite3_open_v2(target.c_str(), &m_dbconn, resultingFlags, nullptr) != SQLITE_OK)

```

Comment:
Obviously the right answer is that we need to put a check higher up in the stack to ensure that the file actually exists, as this returns a very unhelpful error message. (This should be kept.)